plugins/metric_links/data/database.sql

```diff
@@ -1,22 +0,0 @@
-SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
-SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
-SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
-
-
--- -----------------------------------------------------
--- Table `metric_links_status`
--- -----------------------------------------------------
-CREATE TABLE IF NOT EXISTS `metric_links_status` (
```

Comment:
fixed in https://github.com/mfairchild365/site_master/commit/0b90687929919d156565aa6ba47e12914d50e101